hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -830,7 +832,7 @@ message LookupKeyResponse {
 
 message RenameKeyRequest{
     required KeyArgs keyArgs = 1;
-    required string toKeyName = 2;
+    optional string toKeyName = 2;
```

Comment:
why do we need to change toKeyName from required to optional?

Reply:
This place is useless for modification, I will restore this place.